openmetrics/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Are there no default metrics collected with the check ?

Reply:
No it's like prometheus everything will be custom metrics https://github.com/DataDog/integrations-core/blob/master/prometheus/metadata.csv